api/README.md

```diff
@@ -41,10 +41,10 @@ API to allow technical users to query the database and extract information in re
   GET `/earthquakes/date?start_date=01-01-2001&end_date=02-02-2002&sort=descending`
 
 ### 5️⃣ Get Earthquakes by Alert Level
-- 🛠️ **Endpoint**: `GET /earthquakes/alert`
+- 🛠️ **Endpoint**: `GET /earthquakes/alert/color`
```

Comment:
just to be nitpicky - could this be UK spelling for consistency, since we have used UK spelling elsewhere in READMEs?